pages/history.js

```diff
@@ -1,10 +1,92 @@
+import {
+	faBirthdayCake,
+	faCircleNotch,
+	faMask,
+	faRandom,
+	faVideo,
+} from "@fortawesome/free-solid-svg-icons";
 import Head from "next/head";
 import styled from "styled-components";
+import Event from "../components/event";
+
+const EventsContainer = styled.div`
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+
+	& > * {
+		flex: 0 1 calc(50% - 16px);
+		margin: 8px;
+
+		max-width: 35%;
+	}
+
+	@media (max-width: 800px) {
+		flex-direction: column;
+
+		& > * {
+			flex-basis: 100%;
+			max-width: unset;
+		}
+	}
+`;
 
 export default () => (
 	<>
 		<Head>
 			<title>History - The Snakeroom</title>
 		</Head>
+		<EventsContainer>
+			<Event icon={faMask} name="Imposter" time="April 2020">
+				In this event, players receive four messages by other players
+				and one from the bot, and must determine which one was generated
+				by the bot. The third iteration of the Sneknet collects results
+				from each round to determine which of the five messages was
+				generated by the bot. With enough data, the correct answer can
+				automatically be selected. correct answer is known.
```

Comment:
lol what's up with this

```suggestion
				automatically be selected.
```